product_docs/docs/pgd/5/reference/functions-internal.mdx

```diff
@@ -93,20 +93,19 @@ This is an internal function, mainly used by HARP.
 
 ### `bdr.decode_message_payload`
 
-PGD message payload function that decodes the consensus payloads to a more human-readable output.Used primarily by the `bdr.global_consensus_journal_details` debug view.
+PGD message payload function that decodes the consensus payloads to a more human-readable output. Used primarily by the `bdr.global_consensus_journal_details` debug view.
 
 ### `bdr.decode_message_response_payload`
 
 PGD message payload function that decodes the consensus payloads to a more human-readable output. Used primarily by the `bdr.global_consensus_journal_details` debug view.
-
+<!-- This description is identical to the one preceding >
```

Comment:
This description is identical to the one preceding. Is there any differentiation?

Reply:
Yes, fixed.